scripts/core/exp-usm-p2p.yml

```diff
@@ -24,9 +24,9 @@ name: $x_exp_peer_info_t
 typed_etors: True
 etors:
     - name: UR_PEER_ACCESS_SUPPORTED
-      desc: "[uint32_t] 1 if P2P access is supported otherwise P2P access is not supported."
+      desc: "[int] 1 if P2P access is supported otherwise P2P access is not supported."
```

Comment:
nit: maybe this should be int32_t to be more precise?

Reply:
> nit: maybe this should be int32_t to be more precise?

I purposely chose `int`, because I don't think there is any reason that non-32 bit integers can't be used (if the platform uses e.g. 16 bits for `int`). This matches what cuda and hip do, see e.g. https://docs.nvidia.com/cuda/cuda-driver-api/group__CUDA__PEER__ACCESS.html#group__CUDA__PEER__ACCESS_1g4c55c60508f8eba4546b51f2ee545393

The runtime correspondingly retrieves value of size(int): see https://github.com/intel/llvm/pull/15873/files#diff-e5a5ec910bab410f31617da9dfb56ee9e435769e7d6a830e85094c07b80c5454R250

Reply:
Makes sense. Thanks.